src/middlewares/social/social.js

```diff
@@ -0,0 +1,35 @@
+export default (req, res, next) => {
```

Comment:
@Ramadhan0  please add documentation of this function. But also, I don't think the name of this file describes well what is happening inside